applications/nrf_desktop/bootloader_dfu.rst

```diff
@@ -72,27 +72,45 @@ See :ref:`nrf_desktop_memory_layout` for details.
+* In the sysbuild configuration:
+
+  * ``SB_CONFIG_SECURE_BOOT_SIGNING_KEY_FILE`` - Required for providing the signature key file used for image signing and verification.
+    If this Kconfig option does not specify the signature key file, automatically generated signature key files will be used by default.
+
+* In the application configuration:
+
+  * :kconfig:option:`CONFIG_FW_INFO` - Required for the application versioning information.
+  * :kconfig:option:`CONFIG_FW_INFO_FIRMWARE_VERSION` - Set value of this Kconfig option to update application version.
+    The nRF Desktop application with the B0 bootloader configuration builds two application images: one for the S0 slot and the other for the S1 slot.
+    To generate the DFU package, you need to update this configuration only in the main application image as the ``s1_image`` image mirrors it.
+
+.. note::
+    To ensure that update image will be booted, the generated update image's version number must be greater than the version number of application image running on device.
+    Otherwise the update image can be rejected by the bootloader.
 
 .. _nrf_desktop_configuring_mcuboot_bootloader:
 
 Configuring the MCUboot bootloader
 ==================================
 
-To enable the MCUboot bootloader, select the :kconfig:option:`CONFIG_BOOTLOADER_MCUBOOT` Kconfig option in the application configuration.
+To enable the MCUboot bootloader, select the ``SB_CONFIG_BOOTLOADER_MCUBOOT`` Kconfig option in the sysbuild configuration.
 
-The MCUboot private key path (:kconfig:option:`CONFIG_BOOT_SIGNATURE_KEY_FILE`) must be set only in the MCUboot bootloader configuration file.
+The MCUboot private key path (``SB_CONFIG_BOOT_SIGNATURE_KEY_FILE``) must be also set in the sysbuild configuration.
 The key is used both by the build system (to sign the application update images) and by the bootloader (to verify the application signature using public key derived from the selected private key).
+If this Kconfig option is not overwritten in the sysbuild configuration, debug signature key files located in the MCUboot bootloader repository will be used by default.
+
+To set specific version of the application, use the :kconfig:option:`CONFIG_MCUBOOT_IMGTOOL_SIGN_VERSION` Kconfig option in the application configuration.
+If the nRF Desktop application is configured with the MCUboot in the direct-xip mode, the build system builds two application images: one for the primary slot and the other for the secondary slot named ``mcuboot_secondary_app``.
+You need to update this configuration only in the main application image as the ``mcuboot_secondary_app`` image mirrors it.
+
+.. note::
+    To ensure that update image will be booted, the generated update image's version number must be greater than the version number of application image running on device.
+    Otherwise the update image can be rejected by the bootloader.
```

Comment:
IIRC This applies only to MCUboot with XIP (not to MCUboot with swap). It may be worth to mention it.